config/rootfs/debos/scripts/bullseye-libcamera.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# No libuevent, it's needed only for hotplug detection
+# No libboost or raspberrypi pipeline, should be added if tested on raspberrypi
+
+BUILD_DEPS="\
+      g++ \
+      ninja-build \
+      meson \
+      python3-yaml \
+      python3-ply \
+      python3-jinja2 \
+      python3-setuptools \
+      libgnutls28-dev \
+      openssl \
+      pkg-config \
+      libgtest-dev \
+"
+
+apt-get install --no-install-recommends -y  ${BUILD_DEPS}
+
+BUILDFILE=/test_suites.json
+echo '{  "tests_suites": [' >> $BUILDFILE
+
+# Build libcamera and lc-compliance
+########################################################################
+
+LIBCAMERA_URL=https://gitlab.collabora.com/nfraprado/libcamera.git
+mkdir -p /tmp/tests/libcamera && cd /tmp/tests/libcamera
+
+# Use my fork until it gets merged
+git clone --single-branch --branch lcc-gtest-v5 --depth=1 $LIBCAMERA_URL .
```

Comment:
@nfraprado I tried to build a rootfs on staging using your commit and it is breaking looking for git in line 36, so probably you need to install it before using. I am not sure if you have access to jenkins (I don't think so), then I will paste some log lines:

`2021/05/28 00:29:53 bullseye-libcamera.sh | Processing triggers for libc-bin (2.31-12) ...

2021/05/28 00:29:53 bullseye-libcamera.sh | /tmp/script/bullseye-libcamera.sh: line 36: git: command not found
2021/05/28 00:29:53 Action `Build testsuite` failed at stage Run, error: exit status 127`

Reply:
Okay, I missed that indeed, thanks for the feedback. I just force-pushed a commit with git included, could you try and see if it works now?